lib/route.js

```diff
@@ -265,13 +265,23 @@ function route(name, model) {
    * Get a list of resources.
    */
   router.get(collectionRoute, function (req, res) {
-    var ids = [];
+    var query = {},
+        schema = _this._schema[name],
+        idParamType = typeof req.query.ids;
 
-    if (typeof req.query.ids === 'string') ids = req.query.ids.split(',');
-    if (typeof req.query.ids === 'object') ids = req.query.ids;
+    if(idParamType === "undefined") {
+        // For each query params copy value for querying adapter
+        _.each(req.query, function(value,key) { if (key in schema) query[key] = value });
+    }
+    else if(idParamType === "string") {
+        query = req.query.ids.split(',');
```

Comment:
shouldn't ids have been provided as id[]=1&id[]2 making split unnecessary? If that's not the case I think it should be as this seems to violate the Rails standard for how to handle arrays in form-url-encoded query strings.